cmd/bundle.go

```diff
@@ -80,14 +80,15 @@ var bundleInfoCmd = &cobra.Command{
 
 var bundleNamespace string
 var sandboxRole string
+var kubeConfig string
 
 var bundleProvisionCmd = &cobra.Command{
 	Use:   "provision <bundle name>",
 	Short: "Provision ServiceBundle images",
 	Long:  `Provision ServiceBundles from a registry adapter`,
 	Args:  cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		runner.RunBundle("provision", bundleNamespace, args[0], sandboxRole, bundleRegistry, args[1:])
+		preRun("provision", args)
```

Comment:
I was surprised to see that a function called `preRun` ends up calling `runBundle` (I wouldn't expect a function called prerun to prerun _AND_ run a bundle). 

I'd suggest alternate names `bundleAction` or `executeBundle` if you're trying to avoid duplicating the function name `runBundle` from runner.